src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/StartupObjectsEnumProvider.cs

```diff
@@ -55,10 +55,10 @@ public StartupObjectsEnumGenerator(Workspace workspace, UnconfiguredProject proj
         public async Task<ICollection<IEnumValue>> GetListedValuesAsync()
         {
             Project project = _workspace.CurrentSolution.Projects.First(p => PathHelper.IsSamePath(p.FilePath, _unconfiguredProject.FullPath));
-            Compilation compilation = await project.GetCompilationAsync();
+            Compilation ?compilation = await project.GetCompilationAsync();
```

Comment:
Question marks go directly _after_ the type. It's part of the type, not the variable.

Reply:
done